magento/pwa-studio
# [PWA-814] Enable shoppers to change locale using a store view switcher

## Description

Adding store switcher component to allow visitor to switch store views and get content translated to respective store view locale.

## Related Issue





https://jira.corp.magento.com/browse/PWA-814

## Acceptance 



### Verification Stakeholders

@jimbo
@sirugh 
@dpatil-magento 
### Specification



### Verification Steps


1. Configure Magento to have more then one store view
2. See store switcher appeared in the PWA header for desktop / Side navigation for mobile screen
3. Use switcher to change store view/locale
4. Verify that data been retrieved according to selected store view

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
